docs/guides/ai-transpiler-passes.mdx

```diff
@@ -102,6 +112,38 @@ The following custom collection passes for Cliffords, Linear Functions and Permu
 
 These custom collection passes limit the sizes of the collected sub-circuits so they are supported by the AI-powered synthesis passes. Therefore, it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
 
+## Hybrid Qiskit-AI circuit transpilation
+
+The `qiskit-ibm-transpiler` offers the option to configure a hybrid pass manager that combines the best of Qiskit and the AI-powered transpiler passes, so you can run a whole transpilation on your own. This feature behaves similarly to the Qiskit `generate_pass_manager` method. A typical way to use `generate_ai_pass_manager` is as follows:
+
+```python
+    from qiskit_ibm_transpiler.ai import generate_ai_pass_manager
+    from qiskit.circuit.library import EfficientSU2
+    from qiskit_ibm_runtime import QiskitRuntimeService
+
+    backend = QiskitRuntimeService().backend("ibm_fez")
+    fez_coupling_map = backend.coupling_map
+
+    su2_circuit = EfficientSU2(101, entanglement="circular", reps=1).decompose()
+
+    ai_transpiler_pass_manager = generate_ai_pass_manager(,
+        coupling_map=fez_coupling_map,
+        ai_optimization_level=3,
+        optimization_level=3,
+        ai_layout_mode=optimize,
```

Comment:
Is `optimize` supposed to be a string literal or a variable? If the latter, it's not defined. Also, can I set `local_mode` here?